client/slack.go

```diff
@@ -208,3 +222,34 @@ func newUser(u *slack.User) user {
 		LastName:  u.Profile.LastName,
 	}
 }
+
+func newReactionEvent(e *slack.ReactionAddedEvent, u *slack.User, itemuser *slack.User) reactionAddedEvent {
+	return reactionAddedEvent{
+		ReactionUser:   newUser(u),
+		ReactionName:   e.Reaction,
+		EventTimestamp: e.EventTimestamp,
+		ItemTimestamp:  e.Item.Timestamp,
+		ItemType:       e.Item.Type,
+		ChannelId:      e.Item.Channel,
```

Comment:
```suggestion
		ItemChannel:      e.Item.Channel,
```
consistent with the type and event